cmd/netobserv-ebpf-agent.go

```diff
@@ -18,6 +18,20 @@ import (
 	_ "net/http/pprof"
 )
 
+func terminateAgent() (ctx context.Context) {
+
+	logrus.Infof("push CTRL+C or send SIGTERM to interrupt execution")
+	ctx, canceler := context.WithCancel(context.Background())
+	// Subscribe to signals for terminating the program.
+	go func() {
+		stopper := make(chan os.Signal, 1)
+		signal.Notify(stopper, os.Interrupt, syscall.SIGTERM)
+		<-stopper
+		canceler()
+	}()
+	return ctx
+}
```

Comment:
why did u add this ? for standalone testing ?

Reply:
This was to reuse the same code block for terminating both netobserv and PCA.